deluge/tests/test_common.py

```diff
@@ -77,6 +77,8 @@ def test_is_infohash(self):
         self.assertTrue(is_infohash('2dc5d0e71a66fe69649a640d39cb00a259704973'))
 
     def test_get_path_size(self):
+        if windows_check():
+            raise unittest.SkipTest('os devnull is different on windows')
         self.assertTrue(get_path_size(os.devnull) == 0)
```

Comment:
So os.devnull is cross-platform so perhaps `get_path_size` needs fixed rather than skip this test

Reply:
in linux if you do os.path.getsize(os.devnull) you get 0 in windows you get error parameter incorrect nul.